packages/base/Section/src/Section/Section.tsx

```diff
@@ -40,9 +36,33 @@ export interface Props extends BaseProps {
   titleSize?: SizeType<'small' | 'medium'>
 }
 
-const useStyles = makeStyles<Theme>(styles, {
-  name: 'PicassoSection',
-})
+const variantMapping: { [K in VariantType]: string | string[] } = {
+  default: '[&>_:last-child:not(:first-child)]:mt-6',
+  bordered: [
+    '[&>_:last-child:not(:first-child)]:mt-6',
+    'border rounded-md border-solid border-gray-300',
+    'p-8 [&_>_:last-child]:pb-0',
+  ],
+  withHeaderBar: 'p-0 rounded-md border border-solid border-gray-400',
+}
+
+const headerMapping: { [K in VariantType]: string | string[] } = {
+  default: 'flex',
+  bordered: 'flex',
+  withHeaderBar: [
+    'flex',
+    'pt-3 pb-3 pl-4 pr-4',
+    'rounded-tl-md rounded-tr-md rounded-br-0 rounded-bl-0',
+    'border-solid border-l-0 border-r-0 border-t-0 border-b border-gray-400',
+    'bg-gray-100',
+  ],
+}
+
+const collapsedHeaderMapping: { [K in VariantType]: string | string[] } = {
```

Comment:
can we do it like that?
```suggestion
const collapsedHeaderMapping:  Record<VariantType, string | string[]> = {
```

Reply:
@dmaklygin We can indeed, but can you please specify what is difference, do we have any guidelines on that? I think it a bit subjective which one to choose (I remember similar question in https://github.com/toptal/picasso/pull/4276#discussion_r1596171548)

Reply:
`Record` is a generic type that is part of TypeScript's standard library. I would utilize common utilities as much as possible

Reply:
@dmaklygin Replaced it with Record, thanks!